src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -550,11 +550,12 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
   />
   <Touch
       Condition=" '$(DesignTimeBuild)' != 'True' "
-      Files="$(_AndroidStampDirectory)_BuildAdditionalResourcesCache.stamp"
+      Files="$(_AndroidStampDirectory)_BuildAdditionalResourcesCache.stamp;$(MonoAndroidIntermediateResourceCache).stamp"
       AlwaysCreate="True"
   />
   <ItemGroup>
     <FileWrites Include="$(_AndroidResourcePathsCache)" Condition=" Exists ('$(_AndroidResourcePathsCache)') " />
+    <FileWrites Include="$(MonoAndroidIntermediateResourceCache).stamp" Condition=" Exists ('$(MonoAndroidIntermediateResourceCache).stamp') " />
```

Comment:
The `Condition` was added because the line above it had one too.

Reply:
We do because there is a condition on the touch


```
<Touch
      Condition=" '$(DesignTimeBuild)' != 'True' "
      Files="$(_AndroidStampDirectory)_BuildAdditionalResourcesCache.stamp;$(MonoAndroidIntermediateResourceCache).stamp"
      AlwaysCreate="True"
  />
```